nondex-common/src/main/java/edu/illinois/nondex/common/NonDex.java

```diff
@@ -68,6 +70,10 @@ public static NonDex getInstance() {
         return globalInstance;
     }
 
+    public SystematicRandom getSysRandom() {
```

Comment:
The getRandom method is private and part of the edu.illinois.nondex.common package, since I need to get systematicRandom in package edu.illinois.nondex.shuffling I added a new public method purely for getting systematicRandom.  If its ok for me to change the getRandom method to public I will do so and remove the getSysRandom method.